robottelo/decorators/__init__.py

```diff
@@ -2,35 +2,13 @@
 import logging
 from functools import wraps
 
-import pytest
 import unittest2
 
 from robottelo.config import settings
 
 LOGGER = logging.getLogger(__name__)
 OBJECT_CACHE = {}
 
-# Test Tier Decorators
-# CRUD tests
-tier1 = pytest.mark.tier1
-# Association tests
```

Comment:
@mshriver we should also update `docs/features/decorators.rst` to reflect this change.
And do we need to change anything in robottelo-ci because of this?

Reply:
There should be any changes or impact to any tools using these, as functionally they are identical

Reply:
ok. 
and I can raise a pr to update `docs/features/decorators.rst` later.